### ISSUE TITLE
Loss curve

### ISSUE THREAD

**Issue by godchengzhihang** [JiahuiYu/generative_inpainting#325]
Hello @JiahuiYu ,
My g_loss, d_loss and l1_loss are as follows. I feel that g_loss and d_loss are fast converging, but my l1_loss feeling is still falling. I saw that the picture I fixed is good and bad, but the overall is not very good.May I continue to train? The convergence of the final model depends on the l1_loss 
![`R9R}6K6_2RO06I1{RFD HQ](https://user-images.githubusercontent.com/55656735/67910177-a5776700-fbbc-11e9-8583-943e8e02768e.png)
![E 5Y63FZUS(QVS_DCQKIX5O](https://user-images.githubusercontent.com/55656735/67910190-b3c58300-fbbc-11e9-8fd4-2e326a29b50f.png)

**Comment by godchengzhihang** [comment]
deepfillv1

**Comment by JiahuiYu** [comment]
The loss loos normal. Please continue training.

**Comment by godchengzhihang** [comment]
Thank you very much for your reply!I want to ask if the training to l1_loss convergence should be fine, right?Even if the GAN loss has converged but l1_loss does not converge, it is necessary to continue training until l1_loss converges, right? @JiahuiYu

**Comment by JiahuiYu** [comment]
Yes. My experience is to continue training until all losses are converged.

**Comment by godchengzhihang** [comment]
thank you very much! @JiahuiYu I am very sorry to bother you again!I looked at my loss curve today and found that global_gloss is suddenly rising. I want to ask if there is any problem with this? As long as I finally converge on all the loss curves, right? The gloss is still rising, indicating that it has not converged. When I was doing a simple task with wgan-gp, I found that his curve was normal to convergence, and my gloss curve was like a state of convergence yesterday.I found out that it was suddenly rising this morning, but my l1_loss has not yet converged. I don't know if the change in this gloss is a problem with the model I trained.

**Comment by godchengzhihang** [comment]
This is my loss curve:
![3`%HLKMPQ`ZEDAR6@GSJE39](https://user-images.githubusercontent.com/55656735/67998592-1c336380-fc94-11e9-9d10-3014fdc2f61b.png)
![I1MX71EL7V83 3H~U0DMVGF](https://user-images.githubusercontent.com/55656735/67998593-1c336380-fc94-11e9-8b17-ead39e2b66c5.png)

**Comment by JiahuiYu** [comment]
Hi no problem - if you have any question, just posted it here. 

For loss, yours looks quite normal. In general, a large batch size (across multiple gpus if memory is not enough) can stabilize the training.

**Comment by godchengzhihang** [comment]
Thank you very much for your prompt reply and help. I will ask for your help when I have another problem.

**Comment by godchengzhihang** [comment]
@JiahuiYu hi,I am very sorry to bother you again!
![1MQ$WG1(GX6DN3Y(VLB(EJ9](https://user-images.githubusercontent.com/55656735/68065599-aea43780-fd66-11e9-9885-5c3c3272a11d.png)
![35UB(LB~734~HP64EDKCZEY](https://user-images.githubusercontent.com/55656735/68065600-aea43780-fd66-11e9-8167-61fb90580352.jpg)
I found that my l1_loss has converged, but g_loss is not in a convergent state, and d_loss is like convergence.I saw that you have previously seen the d_loss curve in tensorbaord, but my g_loss has not yet converged. Is it a problem with my model? In theory, g_loss and d_loss should be convergent at the end. I would like to ask if my change in g_loss indicates that there is a problem with my model. If there is something wrong with my model, I hope I can get your help.
Also, I observed that my l1_loss loss curve is close to convergence, and I am currently reducing the learning rate by 1/10 to continue training to the end. In your experience, is my approach feasible?(My exposure to deep learning computer vision time is still relatively short, At present, it is also a relatively shallow study of image restoration. There are still many places that I don't understand, so I have to interrupt you a lot. Thank you very much for your help! )
